boards/arm/npcx7m6fb_evb/npcx7m6fb_evb_defconfig

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2020 Nuvoton Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SOC_NPCX7M6FB=y
+CONFIG_SOC_SERIES_NPCX7=y
+CONFIG_BOARD_NPCX7M6FB_EVB=y
+CONFIG_CORTEX_M_SYSTICK=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# General Kernel Options
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=15000000
```

Comment:
@MulinChao this is defined as 48MHz in Kconfig.defconfig, I guess you need to have a single define, only.

Reply:
Done.